src/PublicApiGeneratorTests/Class_events.cs

```diff
@@ -1,39 +0,0 @@
-﻿using System;
-using PublicApiGeneratorTests.Examples;
-using Xunit;
-
-namespace PublicApiGeneratorTests
-{
-    public class Class_events : ApiGeneratorTestsBase
-    {
-        [Fact]
-        public void Should_output_event()
```

Comment:
Did you mistakenly lose this test when moving to another file?

Reply:
Oops, thanks for catching!